org.eclipse.mylyn-target/mylyn-e4.26.target

```diff
@@ -71,6 +71,22 @@
 		</location>
 		<location includeDependencyDepth="infinite" includeDependencyScopes="compile,provided,runtime" includeSource="true" missingManifest="generate" type="Maven">
 			<dependencies>
+				<dependency>
+				    <groupId>com.sun.xml.bind</groupId>
+				    <artifactId>jaxb-impl</artifactId>
+				    <version>2.3.3</version>
+				</dependency>
+				<dependency>
+				    <groupId>jakarta.xml.bind</groupId>
+				    <artifactId>jakarta.xml.bind-api</artifactId>
+				    <version>2.3.3</version>
+				</dependency>
+				<dependency>
+				    <groupId>org.glassfish.hk2</groupId>
```

Comment:
Why do we need this one?

Reply:
I have no idea how it's all wired together. Maybe https://www.vogella.com/blog/eclipse-rcp-java-11-jaxb/ will make more sense to you?

Reply:
Yes, true, seems to be required to activate plain java services expected by JAXB impl with OSGi. We can get this bundle from https://download.eclipse.org/tools/orbit/downloads/drops/R20221123021534/ as well.

Reply:
Hmm, when I moved it to orbit, the build breaks with:
Error:  Cannot resolve project dependencies: Error:    Software being installed: org.eclipse.mylyn.commons.feature.group 3.26.0.qualifier Error:    Missing requirement: org.eclipse.mylyn.commons.ui 3.26.0.qualifier requires 'osgi.bundle; org.apache.commons.lang [2.6.0,3.0.0)' but it could not be found Error:    Cannot satisfy dependency: org.eclipse.mylyn.commons.feature.group 3.26.0.qualifier depends on: org.eclipse.equinox.p2.iu; org.eclipse.mylyn.commons.ui 0.0.0

I also tried the 2.3.0 version, but it has similar, but different, problems

Reply:
Tried to change it to commons.lang3 but it's not available from orbit(?)

Reply:
OMG. Let's postpone this improvement, I'd rather have a closer look on our .target later.